src/core/libmaven/mzSample.cpp

```diff
@@ -810,8 +810,7 @@ void mzSample::parseMzXML(const char* filename)
         setInstrumentSettigs(doc, spectrumstore);
         // parse mzXML information from the scan
         parseMzXMLData(spectrumstore);
-    } else
-        throw MavenException(ErrorMsg::ParsemzXml);
```

Comment:
@MdAzmatAli  why is this exception removed??? Wasn't it getting handled somewhere? I see this exception is thrown in other such functions too. This means those would also lead to a crash if the conditions match.

Reply:
@MdAzmatAli Sakshi is right. These exceptions are thrown in a few other places in mzSample.cpp. They should ideally be caught and informed about the user on what we were expecting and what went wrong.

For example, in mzfileio.cpp, we [catch memory allocation exception](https://github.com/ElucidataInc/ElMaven/blob/develop/src/gui/mzroll/mzfileio.cpp#L82-L88) when user runs out of RAM while loading samples. Something similar should happen in this case as well - catch the specific type of exception (`ErrorMsg::ParsemzXml`) and then tell user that something is wrong with the file.